src/axom/sidre/spio/IOManager.cpp

```diff
@@ -167,28 +167,17 @@ void IOManager::write(sidre::Group* datagroup,
     m_baton = new IOBaton(m_mpi_comm, num_files, m_comm_size);
   }
 
+  SLIC_ERROR_IF(m_use_scr && num_files != m_comm_size,
+                "SCR requires a file per process");
+
```

Comment:
It is now possible to read/write shared files with SCR.  To do that, the user needs to configure SCR to read/write files to a global file system, i.e., use cache bypass mode, which is the default.  Under this mode, users don't benefit from caching, but the SCR API is still valid.  We added that feature to improve consistency and portability.

I remember we discussed this before in this context, and left it as possible future work.  Just figured I'd repeat it now to keep that fresh.

Reply:
Yes, I think this should still be left for future work.  We can change this requirement in IOManager at any time.